src/components/Book.js

```diff
@@ -0,0 +1,59 @@
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBookFromApi } from '../redux/books/booksSlice';
+import RemoveButton from './RemoveButton';
+
+/*eslint-disable */
+const BookInfo = ({ book }) => {
+  const dispatch = useDispatch();
+
+  const handleClick = (idBook) => {
+    dispatch(removeBookFromApi(idBook));
+  };
+
+  return (
+    <div className="book-container">
+      <div className="book-item">
+        <p className="book-category">{book.category}</p>
+        <h3 className="book-title">{book.title}</h3>
+        <p className="book-author">{book.author}</p>
+        <ul className="book-actions">
+          <li>Comments</li>
+          <li>
+            <RemoveButton handleRemoveClick={() => { handleClick(book.item_id); }} />
+          </li>
+          <li>Edit</li>
+        </ul>
+      </div>
+       <div className="book-item"> 
+         <div className="book-improvment">
+            <div className="circle">
+            </div>
+            <div className="percentage">
+              <p className="competion-percentage"> 80 % </p> 
+              <p className="completed"> completed </p>
+            </div>
+          </div>
```

Comment:
the book component looks great only this element needs to be aligned to the right also inrease its size 

![Capture d’écran du 2023-08-04 22-26-53](https://github.com/akbarsazish/Bookstore-React/assets/59707859/ecf571f0-035f-4116-a17c-0e003d6ea0fe)
![Capture d’écran du 2023-08-04 22-26-42](https://github.com/akbarsazish/Bookstore-React/assets/59707859/79f5bda9-605e-4a02-894a-d62c49695a9b)